documentation/modules/security/proc-renewing-your-own-ca-certificates.adoc

```diff
@@ -16,92 +17,68 @@ The procedure describes the renewal of CA certificates in PEM format.
 
 * The Cluster Operator is running.
 * xref:installing-your-own-ca-certificates-{context}[Your own CA certificates and private keys are installed].
-* You have new cluster and clients X.509 certificates and keys in PEM format.
-
-These could be generated using an `openssl` command, such as:
-
-[source,shell,subs="+quotes"]
-openssl req -x509 -new -days _NUMBER-OF-DAYS-VALID_ --nodes -out ca.crt -keyout ca.key
+* You have new cluster and clients X.509 certificates in PEM format.
```

Comment:
Should this be `or`? I assume you can replace only one of them since they are completely independent?